projects/storefrontlib/shared/components/list-navigation/sorting/sorting.component.html

```diff
@@ -4,7 +4,6 @@
   placeholder="{{ placeholder }}"
   (change)="sortList($event)"
   [ngModel]="selectedOption"
-  [attr.aria-label]="selectedLabel || placeholder"
```

Comment:
```suggestion
  [inputAttrs]="{'aria-label': selectedLabel || placeholder, 'aria-controls': ariaControls}"
```

Reply:
As we discussed, this does not solve the error reported by AMP a11y testing